docs/data/routes/docs/nextjs/getting-started/walkthrough-dotnetnew/en.md

```diff
@@ -3,4 +3,160 @@ name: walkthrough-dotnetnew
+
+### Install the template
+
+You download and install the template in your file system. Afterwards, you can create solutions based on the template.
+
+To install the template:
+
+1. Open PowerShell with administrator privileges.
+
+2. Run the following command:
+
+   ```powershell
+   dotnet new -i Sitecore.DevEx.Templates --nuget-source https://sitecore.myget.org/F/sc-packages/api/v3/index.json
+   ```
+
+   The `dotnet` tool does not provide obvious feedback that the install was successful. However, the install command ends by listing all of the installed templates. If the install is successful, the list includes `sitecore.aspnet.gettingstarted`.
+
+### Create the MyProject solution
+
+You can name you solution anything you like, but in this example we name the solution `MyProject`.
+
+To create the `MyProject` solution:
+
+1. Open PowerShell with administrator privileges.
+
+2. Check if you have Internet Information Server running on port 443:
+
+   ```powershell
+   Get-Process -Id (Get-NetTCPConnection -LocalPort 443).OwningProcess
+   ```
+
+   If you do, you must to stop it:
+
+   ```powershell
+   iisreset /stop
+   ```
+
+3. Check if you have Apache Solr or any other service running on port 8984:
+
+   ```powershell
+   Get-Process -Id (Get-NetTCPConnection -LocalPort 8984).OwningProcess
+   ```
+
+   If you do, you must stop it:
+
+   ```powershell
+   Stop-Service -Name "<the name of your service>"
+   ```
+
+   Or, if you have started it with Non-Sucking Service Manager, stop it with this command:
+
+   ```powershell
+   nssm stop "<the name of your service>"
+   ```
+
+4. Go to your solutions folder and create the `MyProject` solution and project:
+
+   ```powershell
+   dotnet new sitecore.nextjs.gettingstarted `
+   -n MyProject `
+   --NuGetAuth true `
+   --SitecoreNuGet http://nuget1ca2.dk.sitecore.net/nuget/sc_packages_preview/ `
+   --SecondaryNuGet http://nuget1ca2.dk.sitecore.net/nuget/10.1.0_master/ `
+   --SitecorePSGallery http://nuget1ca2.dk.sitecore.net/nuget/Sitecore_Gallery `
+   --ModulesDockerRegistry do-hrbr-01-dk1.dk.sitecore.net/sxp/modules/ `
+   --ToolsDockerRegistry do-hrbr-01-dk1.dk.sitecore.net/experimental/ `
+   --SXPDockerRegistry do-hrbr-01-dk1.dk.sitecore.net/sxp/ `
+   --allow-scripts yes
+   ```
+
+5. Go to the `MyProject` folder
+
+6. You use the provided `init.ps1` script to prepare the following items for the Sitecore container environment:
+
+   * A valid/trusted wildcard certificate for `*.myproject.localhost`
+   * Hosts file entries for `myproject.localhost`
+   * Required environment variable values in `.env` for the Sitecore instance
+
+   > * Using non-Latin characters in the project/folder name can give unexpected results because of character limitations in Docker registry names and URLs.
+   >
+   > * See the [Sitecore Containers documentation](https://containers.doc.sitecore.com/) for more information on these preparation steps.
```

Comment:
https://doc.sitecore.com/developers/100/developer-tools/en/containers-in-sitecore-development.html

(will this link be different for 10.1?)

Reply:
Yes, updating.